beer_and_tell/2016-08-August.md

```diff
@@ -0,0 +1,111 @@
+Once a month, web developers from across the Mozilla Project get together to
+talk about our side projects and drink, an occurrence we like to call "Beer and
+Tell".
+
+There's a [wiki page available][wiki] with a list of the presenters, as well as
+links to their presentation materials. There's also a [recording
+available][recording] courtesy of Air Mozilla.
+
+[wiki]: https://wiki.mozilla.org/Webdev/Beer_And_Tell/2016/August
+[recording]: https://air.mozilla.org/webdev-beer-and-tell-august-2016/
+
+## Osmose: PyJEXL
+First up was [Osmose][] (that's me!), who shared [PyJEXL][], an implementation
+of [JEXL][] in Python. JEXL is an expression language based on JavaScript that
+computes the value of an expression when applied to a given context. For
+example, the statement `2 + foo`, when applied to a context where `foo` is 7,
+evaluates to 9. JEXL also allows for defining custom functions and operators to
+be used in expressions.
+
+JEXL is implemented as a JavaScript library; PyJEXL allows you to parse and
+evaluate JEXL expressions in Python, as well as analyze and validate
+expressions.
+
+[Osmose]: https://mozillians.org/en-US/u/Osmose/
+[PyJEXL]: https://github.com/Osmose/pyjexl
+[JEXL]: https://github.com/TechnologyAdvice/Jexl
+
+## pmac: rtoot.org
+Next up was [pmac][], who shared [rtoot.org][], the website for "The Really
+Terrible Orchestra of the Triangle". It's a static site for pmac's local
+orchestra built using [lektor][], an easy-to-use static site generator. One of
+the more interesting features pmac highlighted was lektor's [data model][]. He
+showed a lektor plugin he wrote that added support for CSV fields as part of the
```

Comment:
Please link `lektor plugin` to https://github.com/pmac/rtoot/blob/master/packages/csv-field/lektor_csv_field.py